packages/uniforms/__suites__/render.tsx

```diff
@@ -1,43 +1,30 @@
 import { render as renderOnScreen, RenderResult } from '@testing-library/react';
 import React, { ReactElement, cloneElement } from 'react';
 import SimpleSchema, { SimpleSchemaDefinition } from 'simpl-schema';
-import { BaseForm, Context, UnknownObject, context, randomIds } from 'uniforms';
+import { Context, UnknownObject } from 'uniforms';
 import { SimpleSchema2Bridge } from 'uniforms-bridge-simple-schema-2';
+import { AutoForm } from 'uniforms-unstyled';
 
-const randomId = randomIds();
 export function render<P, Model extends UnknownObject>(
   element: ReactElement<P>,
   schema?: SimpleSchemaDefinition,
-  contextValueExtension?: Partial<Context<Model>>,
+  contextValueExtension?: Pick<Partial<Context<Model>>, 'onChange'>,
```

Comment:
`contextValueExtension` is no longer needed. It is used only for the `onChange` function. Just create the `onChange` param.

The whole `render` could also be refactored. `schema`, `onChange` and `model` are optional. They can be moved to an object param. We could avoid these kind of uses:

```
render(
  <ListField name="x">
    <Child />
    PlainText
  </ListField>,
  { x: Array, 'x.$': String },
  undefined,
  { x: [undefined, undefined] },
);
```

However, this is a big refactor because we use `render` many times, so maybe we can do it in another PR.

Reply:
Yep, it is too much for this PR (for me). But if you want to refactor, you can commit to this PR.

Reply:
Let's do this in another PR. Maybe just write `TODO` comment.